doc/build/changelog/unreleased_14/4428.rst

```diff
@@ -0,0 +1,8 @@
+.. change::
+    :tags: bug, fairly easy, orm
```

Comment:
**Federico Caselli** (CaselIT) wrote:

I believe the tag fairly easy does not exists

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2099

Reply:
Thanks @CaselIT  i have now removed the tag.